src/Data/Fin/Induction.agda

```diff
@@ -59,6 +60,20 @@ open WF public using (Acc; acc)
   induct {suc i} (acc rec) = Pᵢ⇒Pᵢ₊₁ i (induct (rec (inject₁ i) i<i+1))
     where i<i+1 = ℕ<⇒inject₁< (i<1+i i)
 
+≤-Induction : ∀ (P : Pred (Fin (suc n)) ℓ) {i} →
+              P i →
+              (∀ {i} → P (inject₁ i) → P (suc i)) →
+              ∀ {j} → j ≥ i → P j
+≤-Induction P {i} Pi Pᵢ⇒Pᵢ₊₁ {j} j≥i = induct (<-wellFounded _) (<-cmp i j) j≥i
+  where
+  induct : ∀ {j} → Acc _<_ j → Tri (i < j) (i ≡ j) (i > j) → j ≥ i → P j
+  induct {suc j} (acc rs) (tri< (s≤s i≤j) _ _) _ = Pᵢ⇒Pᵢ₊₁ finB
```

Comment:
Very minor, but can we put this case with the additional `where` clause last? Makes it easier to read I think.

Reply:
I put it in the last case.